Private/Utilities.ps1

```diff
@@ -34,17 +34,31 @@ url = https://github.com/3shapeAS/dockerbuild-pwsh.git
     $configData | Out-File -FilePath "$dotGitPath/config" -Encoding ascii
 }
 
-function Add-Postfix {
+function Add-RegistryPostfix {
```

Comment:
Not sure why you want to change the name? The intention behind the function didn't change as I see it.

Reply:
Because this method is specific to adding postfix to registry value. Generalizing it will cause confusion for those who aren't aware of its original intention.

Reply:
Also added logics/checks in the method are very specific to verifying that if the value parsed ends with the postfix. I'd like this method to be special purposed, that i do not need to do checks wherever it is used.